app/views/content/service_privacy_notice_candidate.html.erb

```diff
@@ -113,6 +113,10 @@
     <p class="govuk-body">If you’re successfully recruited to a teacher training course, your data will also be shared
       between Apply and Register trainee teachers, which are both delivered by the Department for Education. This is
       to allow you to be registered as a trainee teacher for the start of your teacher training course.</p>
+    
+    <p class="govuk-body">If you’re successfully recruited to a teacher training course, your data will also be shared between Apply and Teaching Vacancies, which are both delivered by the Department for Education.
```

Comment:
Can we be more explicit about which data will be shared? Is it just name and email?